lib/fathead/arrow_docs/parse.py

```diff
@@ -0,0 +1,57 @@
+"""Parse Arrow docs."""
+
+from bs4 import BeautifulSoup
+import codecs
+import re
+
+INFILE = 'download/docs.html'
+OUTFILE = 'output.txt'
+
+abstract_fmt = '<section class="prog__container"><p>{}{}</p></section>'
```

Comment:
@danidee10 the `<p>` tag should not include the code as well. The `<p>` and `<pre><code>` should be at the same level:

```html
<section>
  <p>$text</p>
  <pre>
    <code>$code</code>
  </pre>
</section>
```

When there's no code, there should just be `<section><p>$text</p></section>` 👍